src/attackmate/variablestore.py

```diff
@@ -72,6 +72,20 @@ def remove_sign(self, name: str, sign: str = '$') -> str:
         else:
             return name
 
+    def get_list(self, listname: str) -> list[str]:
+        name = self.remove_sign(listname)
+        if name in self.lists:
+            return self.lists[name]
+        else:
+            raise VariableNotFound
+
+    def get_str(self, variable: str) -> str:
+        name = self.remove_sign(variable)
+        if name in self.variables:
+            return self.variables[name]
+        else:
+            raise VariableNotFound
+
```

Comment:
duplicate, maybe use a common function in both get_list and get_str that removes sign and retrieves from a dict store /raises exception 
```
def get_item(self, store: dict, name: str):
    name = self.remove_sign(name)
    try:
        return store[name]
    except KeyError:
        raise VariableNotFound
```